tableauserverclient/server/endpoint/endpoint.py

```diff
@@ -1,4 +1,4 @@
-from .exceptions import ServerResponseError, InternalServerError
+from .exceptions import ServerResponseError, InternalServerError, NonXMLResponseError
```

Comment:
Can you add a test for the parse error explicitly, so that this missing import is tested?

Reply:
Done!